modules/database/src/main/resources/database.properties

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###############################################################################
-database.url.host=${mysql.host}
-database.url.port=${mysql.port}
-database.url.dbname=${mysql.dbName}
-database.url.args=${mysql.Args}
+database.url.host:${mysql.host}
```

Comment:
either way ... i tend to prefer ':' but that's a nit.

defer given we should fix this app-wide ... we can always issue subsequent patches.